erpnext/manufacturing/doctype/work_order/work_order_calendar.js

```diff
@@ -2,13 +2,11 @@
 // For license information, please see license.txt
 
 frappe.views.calendar["Work Order"] = {
-	fields:  ["planned_start_date", "planned_end_date", "status", "produced_qty", "qty", "name", "name"],
 	field_map: {
 		"start": "planned_start_date",
 		"end": "planned_end_date",
 		"id": "name",
-		"title": "name",
-		"status": "status",
```

Comment:
Why are we removing this?

Reply:
Because the bloomstack customization overrides this via custom app